ga4gh/backend.py

```diff
@@ -446,137 +455,148 @@ def _noObjectGenerator(self):
-            include = True
-            if request.name:
-                if request.name != obj.getLocalId():
-                    include = False
-            if request.individual_id:
-                if request.individual_id != obj.getIndividualId():
-                    include = False
-            if include:
-                results.append(obj)
-        return self._objectListGenerator(request, results)
-
-    def individualsGenerator(self, request):
-        dataset = self.getDataRepository().getDataset(request.dataset_id)
-        results = []
-        for obj in dataset.getIndividuals():
-            include = True
-            if request.name:
-                if request.name != obj.getLocalId():
-                    include = False
-            if include:
-                results.append(obj)
-        return self._objectListGenerator(request, results)
-
-    def readGroupSetsGenerator(self, request):
+        query = self._registry_db.get_datasets_search_query(request)
+        self._run_sql_query(request, query, responseBuilder)
+
+    def bioSamplesGenerator(self, request, responseBuilder):
+        # Ensure that the dataset exists.
+        self._registry_db.get_dataset(request.dataset_id)
+        query = self._registry_db.get_bio_samples_search_query(request)
+        self._run_sql_query(request, query, responseBuilder)
+
+        # dataset = self.getDataRepository().getDataset(request.dataset_id)
+        # results = []
+        # for obj in dataset.getBioSamples():
+        #     include = True
+        #     if request.name:
+        #         if request.name != obj.getLocalId():
+        #             include = False
+        #     if request.individual_id:
+        #         if request.individual_id != obj.getIndividualId():
+        #             include = False
+        #     if include:
+        #         results.append(obj)
+        # return self._objectListGenerator(request, results)
+
+    def individualsGenerator(self, request, responseBuilder):
+        # Ensure that the dataset exists.
+        self._registry_db.get_dataset(request.dataset_id)
+        query = self._registry_db.get_individuals_search_query(request)
+        self._run_sql_query(request, query, responseBuilder)
+        # dataset = self.getDataRepository().getDataset(request.dataset_id)
+        # results = []
+        # for obj in dataset.getIndividuals():
+        #     include = True
+        #     if request.name:
+        #         if request.name != obj.getLocalId():
+        #             include = False
+        #     if include:
+        #         results.append(obj)
+        # return self._objectListGenerator(request, results)
+
+    def readGroupSetsGenerator(self, request, responseBuilder):
+        # """
+        # Returns a generator over the (readGroupSet, nextPageToken) pairs
+        # defined by the specified request.
+        # """
+        # dataset = self.getDataRepository().getDataset(request.dataset_id)
```

Comment:
you probably already know this, but a quick way to comment out blocks of code is just to turn it into a triple-quoted string